pkg/test/resourcefixture/testdata/basic/vertexai/v1beta1/vertexaiindex/update.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: vertexai.cnrm.cloud.google.com/v1beta1
+kind: VertexAIIndex
+metadata:
+  labels:
+    label-one: "value-one"
+  name: vertexaiindex-${uniqueId}
+spec:
+  displayName: "vertex AI index"
+  region: us-central1
+  description: "an example vertex AI index"
+  metadata:
+    contentsDeltaUri: ${KCC_VERTEX_AI_INDEX_TEST_DATA_URI}
```

Comment:
Do you think it's reasonable that we update this field to verify the special handling?

Reply:
Do you mean we set `contentsDeltaUri` to "${KCC_VERTEX_AI_INDEX_TEST_DATA_URI}" in create.yaml and change it to something like "${KCC_VERTEX_AI_INDEX_TEST_DATA_URI_2}" in update.yaml? We can do it but not sure if it helps verifying our patch.

Reply:
Discussed offline. It looks like there is no good way for us to identify whether `isCompleteOverwrite` is configured or not so updating this field may not be necessary in the dynamic test.

Though it might be a good idea to verify it via the VCR testing in the future.